Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:18
+
+# Set working directory
+WORKDIR /src
+
+# Install dependencies
+COPY package*.json ./
+RUN npm install
+
+# Add node_modules bin to PATH
+ENV PATH /app/node_modules/.bin:$PATH
```

Comment:
I don't see any reference to `/app`. I believe (though I may be wrong) since the working directory is `/src` you'll find the `node_modules` in there. Another note, it's a nice feature to add node_modules to path. But I've also felt comfortable just telling folks to `npx` modules.

Reply:
@axecopfire You're correct; the working directory is /src, not /app. I'll update the PATH accordingly. As for adding node_modules to the PATH, it's to nice-to-have but not essential. We can use npx to execute local binaries if that's more straightforward.